sdk/translation/azure-ai-translation-document/src/main/java/com/azure/ai/translation/document/models/DocumentStatusResult.java

```diff
@@ -165,14 +167,12 @@ public String getTo() {
     }
 
     /**
-     * Get the error property: This contains an outer error with error code, message, details, target and an
-     * inner error with more descriptive details.
+     * Get the ResponseError property: This contains error code and message.
      *
-     * @return the error value.
+     * @return the ResponseError value.
      */
-    @Generated
-    public TranslationError getError() {
-        return this.error;
+    public ResponseError getError() {
+        return new ResponseError(this.error.getCode().toString(), this.error.getMessage());
```

Comment:
Also, we should not create a new ResponseError object everytime `getError` method is called. It should be created only the first time and the same instance should be reused for subsequent calls.

Reply:
Updated